docs/hopsworksai/aws/restrictive_permissions.md

```diff
@@ -141,17 +144,113 @@ If you want to learn more about how this policy works check out:
-VPC during instance configuration. Selecting any other VPCs or instance profiles will result in permissions errors.
+You can now create a new Hopsworks instance in Hopsworks.ai by selecting the configured instance profile,
+VPC and security group during instance configuration. Selecting any other VPCs or instance profiles will result in permissions errors.
 
-## Step 5: Supporting multiple VPCs
+### Step 5: Supporting multiple VPCs
 
 The policy can be extended to give Hopsworks.ai access to multiple VPCs.
 See: [Creating a Condition with Multiple Keys or Values](https://docs.aws.amazon.com/IAM/latest/UserGuide/reference_policies_multi-value-conditions.html).
+
+### Backup permissions
+
+The following permissions are only needed for the backup feature.You can remove them if you are not going to create backups or if you do not have access to this Enterprise feature.
+
+```json
+      {
+        "Sid": "EC2BackupCreation",
+        "Effect": "Allow",
+        "Action": [
+          "ec2:CreateImage",
+	        "ec2:DescribeImages",
+	        "ec2:DeregisterImage",
+	        "ec2:DeleteSnapshot",
+	        "ec2:DescribeSnapshots"
+        ],
+        "Resource": "*",
+      }
+```
+
+### Other removable permissions
+
+The following permissions are needed to give an early warning if your VPC and security groups are badly configured. You can remove them if you don't need this early warning.
+```json
+          "ec2:DescribeVpcAttribute",
+          "ec2:DescribeRouteTables"
+```
+The following permissions are used to let you close and open ports on your cluster from hopswork.ai, you can remove them if you do not want to open ports on your cluster or if you want to manually open ports in EC2.
+
+```json
+      {
+        "Sid": "EC2VpcNonResourceSpecificActions",
+        "Effect": "Allow",
+        "Action": [
+          "ec2:AuthorizeSecurityGroupIngress",
+          "ec2:RevokeSecurityGroupIngress",
+          "ec2:DeleteSecurityGroup"
+        ],
+        "Resource": "*",
+        "Condition": {
+          "ArnLike": {
+            "ec2:vpc": "arn:aws:ec2:*:*:vpc/[VPC_ID]"
+          }
+        }
+      }
+```
+
+## Limiting the instance profile permissions
+
+If you do not intend to take backups or if you do not have access to this Enterprise feature you can remove the permissions that are only used by the backup feature when [configuring instance profile permissions](../getting_started/#step-2-creating-instance-profile) .
+For this remove the following permissions from the instance profile:
+
+```json
+      "S3:PutLifecycleConfiguration", 
+      "S3:GetLifecycleConfiguration", 
+      "S3:PutBucketVersioning", 
+      "S3:GetBucketVersioning"
+```
+
+Hopsworks put its logs in cloudwatch so that you can access them without having to ssh into the machine. If you are not interested in this feature you can remove the following from your instance profile policy:
```

Comment:
```suggestion
Hopsworks put its logs in Amazon CloudWatch so that you can access them without having to ssh into the machine. If you are not interested in this feature you can remove the following from your instance profile policy:
```